recipe/bld.bat

```diff
@@ -34,11 +34,46 @@ cmake -G "Ninja" ^
       -DGDAL_USE_ARROWDATASET=OFF ^
       -DOGR_REGISTER_DRIVER_ARROW_FOR_LATER_PLUGIN=ON ^
       -DOGR_REGISTER_DRIVER_PARQUET_FOR_LATER_PLUGIN=ON ^
+      -DOGR_REGISTER_DRIVER_OPENJPEG_FOR_LATER_PLUGIN=ON ^
```

Comment:
openjpeg is a raster driver, so the CMake options for it are all ``GDAL_`` prefixed instead of ``OGR_`` (several places to change in this commit)